uefi/src/table/cfg.rs

```diff
@@ -26,26 +30,61 @@ pub struct ConfigTableEntry {
     /// Whether this is a physical or virtual address depends on the table.
     pub address: *const c_void,
 }
-/// Entry pointing to the old ACPI 1 RSDP.
```

Comment:
To make the transition to the new constants easier, how about keeping the ones here but marking them as `deprecated`, with the deprecation text saying to use `ConfigTableEntry::*` constants instead.